libs/@guardian/libs/package.json

```diff
@@ -27,6 +27,9 @@
 		"tsc": "wireit",
 		"verify-dist": "wireit"
 	},
+	"dependencies": {
+		"@guardian/ophan-tracker-js": "2.2.10"
+	},
```

Comment:
this should also be added to the `peerDependencies` too

rationale here: https://github.com/guardian/recommendations/blob/main/npm-packages.md#peerdependencies

Reply:
I'm not sure, but advice would be appreciated!